src/Sylius/Behat/Page/ElementNotFoundException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class ElementNotFoundException extends \Exception
```

Comment:
No need to declare it final. It's handy to allow creating e.g. exception `DeleteButtonNotFoundException extends ElementNotFoundException`